core/idl/wallet/common/wallet.djinni

```diff
@@ -333,8 +333,17 @@ TimePeriod = enum {
 # Class representing an account.
 Account = interface +c {
     # Key of the synchronization duration time in the synchronize event payload.
-    # The value is stored in a int 64 time expressed in miliseconds.
+    # The value is stored in a int64 time expressed in miliseconds.
     const EV_SYNC_DURATION_MS: string = "EV_SYNC_DURATION_MS";
+    # Indicates the last block height fetch during synchronization
+    # The value is stored as a int32
+    const EV_SYNC_LAST_BLOCK_HEIGHT: string = "EV_SYNC_LAST_BLOCK_HEIGHT";
+    # Indicates the number of new operations during the synchronization
+    # The value is stored as a int32
+    const EV_SYNC_NEW_OPERATIONS: string = "EV_SYNC_NEW_OPERATIONS";
```

Comment:
When is this message been sent? Is it once per sync?

Reply:
Exactly, at the end of the sync